packages/theme/composables/useUrlResolver/index.ts

```diff
@@ -24,15 +24,15 @@ export function useUrlResolver(): UseUrlResolverInterface {
     search: null,
   });
 
-  const search = async (): Promise<EntityUrl | {}> => {
+  const search = async (): Promise<ROUTE_TYPE> => {
     loading.value = true;
-    let results: EntityUrl = {};
+    let results = null;
```

Comment:
```suggestion
    let results : EntityUrl | null = null;
```
Don't remove the type assertion. Right now the type of results is "any"

Reply:
If EntityUrl doesn't work now please use something else that fits

Reply:
fixed, but I had a problem with wrong generic type and I needed to add @ts-ignore in one line below.

Reply:
Wouldn't count ts-ignore as a fix unless there's some third party TS mistake :P Hopefully Bart's comment about removing the generic should help

Reply:
After chatting with Bartek, the generic type need to stay here. I tried to fix ts-ignore but without success. Could you check out my branch and check that by yourself?